frontend/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@bandwidth/webrtc-browser": "^0.10.0",
+    "@bandwidth/webrtc-browser": "^0.13.1",
```

Comment:
There should be changes to the lock file as well. Did you run `npm update "@bandwidth/webrtc-browser"`?